### PR TITLE
Add a hub for bitwarden clients

### PR DESCRIPTION
**Pull request by nono**
It allows the client to listen with a websocket, and the server can send them notifications about ciphers and folders.

### CODE REVIEW

File: web/bitwarden/hub.go [hunk 1]
```diff
@@ -216,6 +264,75 @@ func buildNotification(e *realtime.Event, userID string) *notification {
 	return &msg
 }
 
+func buildFolderPayload(e *realtime.Event, userID string) map[string]interface{} {
+	var updatedAt interface{}
+	var date string
+	if doc, ok := e.Doc.(*couchdb.JSONDoc); ok {
```
Comment (taratatach) [inline web/bitwarden/hub.go]: Realtime events can contain documents in another format than `realtime.JSONDoc`?

File: web/bitwarden/bitwarden.go [hunk 2]
```diff
@@ -44,11 +44,11 @@ func GetProfile(c echo.Context) error {
 			"error": "invalid token",
 		})
 	}
-	settings, err := settings.Get(inst)
+	setting, err := settings.Get(inst)
```
Comment (taratatach) [inline web/bitwarden/bitwarden.go]: How come the linter didn't error out on this reassignment of `settings`?

File: web/bitwarden/hub.go (hunk 1 above)
Comment (nono) [reply web/bitwarden/hub.go]: Yes, it's a bit of technical debt in the realtime, but it's hard to fix. The documents in realtime must match the `realtime.Doc` interface, and, in practice, it depends on how the update was made, and if we are using memory (self-hosted and development image) or redis (production with several cozy-stack servers).

Comment (taratatach) [reply web/bitwarden/hub.go]: If we have an interface then could we use functions to access those attributes in a standardized fashion?

File: web/bitwarden/bitwarden.go (hunk 2 above)
Comment (nono) [reply web/bitwarden/bitwarden.go]: I don't see a linter that checks that on https://github.com/golangci/awesome-go-linters

File: web/bitwarden/hub.go (hunk 1 above)
Comment (nono) [reply web/bitwarden/hub.go]: Well, I don't see a convenient way to do that

**Pull request merged**